modules/raft/src/main/java/org/apache/ignite/raft/jraft/util/timer/DefaultTimer.java

```diff
@@ -91,7 +91,7 @@ public boolean isCancelled() {
                 @Override
                 public boolean cancel() {
                     final ScheduledFuture<?> f = future;
-                    return f != null && f.cancel(true);
+                    return f != null && f.cancel(false); // Avoid interrupting thread in the pool.
```

Comment:
Why we need this change?

Reply:
To avoid interruption of threads in the pool (which is shared)
Produces side effects like abnormal timer task termination (for example, caused by ClosedByInterruptException)